shared/util/open-url.desktop.js

```diff
@@ -1,5 +1,9 @@
 import {shell} from 'electron'
 
 export default function openURL (url) {
+  if (!url) {
```

Comment:
The bug with `profile.proof` being left undefined was failing silently, this'll make it easier to notice.